questionnaire/questionnaire-service.js

```diff
@@ -47,6 +49,25 @@ function createQuestionnaireService({
         const uuidV4 = uuidv4();
         const questionnaire = templates[templateName](uuidV4);
 
+        if (apiVersion === '2023-05-17') {
+            if (!owner) {
+                throw new VError(
+                    {
+                        name: 'OwnerNotFound'
+                    },
+                    `Owner data must be defined`
+                );
+            }
+
+            questionnaire.answers = {
+                ...questionnaire.answers,
```

Comment:
This is the first ever answers in the questionnaire lifecycle, so we can drop the spread